site/gatsby-site/i18n/locales/es/login.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Login": "Ingresar",
+  "Sign up": "Registrarse",
+  "Log out": "Salir",
+  "Password": "Contraseña",
+  "Confirm password": "Confirmar contraseña",
+  "Forgot password?": "¿Olvidó su contraseña?",
+  "Login with Facebook": "Ingresar con Facebook",
+  "Login with Google": "Ingresar con Google",
+  "Sign up with Facebook": "Registrarse con Facebook",
+  "Sign up with Google": "Registrarse con Google",
+  "Logged in as ": "Ingresado como ",
+  "Logging you out...": "Saliendo...",
+  "Succesfully sent password reset email": "Se ha enviado un correo para restablecer la contraseña",
+  "Account created": "Cuenta creada",
+  "Enter new password": "Ingrese la nueva contraseña",
+  "Confirm new password": "Confirme la nueva contraseña",
+  "Succesfully updated password, click here to ": "Se ha actualizado la contraseña, haga clic aquí para ",
+  "Don't have an account?": "¿No tiene una cuenta?",
```

Comment:
Not sure if we should use formal or informal language (yo uso tuteo 🤔 )

Reply:
I think tuteo is fine if we all agree on that.